libqtile/widget/launchbar.py

```diff
@@ -122,6 +127,11 @@ def __init__(
 
     def _configure(self, qtile, pbar):
         base._Widget._configure(self, qtile, pbar)
+
+        if not (has_xdg or self.text_only):
```

Comment:
You don't need `pyxdg` to have icons. Users can still use icons by passing a path as the first value. All `pyxdg` does is find the icon, it's not relevant to displaying it.

Reply:
Done! Let me know if anything else needs to be done.